sample-app/build.gradle

```diff
@@ -1,7 +1,8 @@
 buildscript {
     repositories { jcenter() }
     dependencies { 
-        classpath 'com.android.tools.build:gradle:0.12.+' // 0.12.x is for Gradle 1.12 projects
+        classpath 'com.android.tools.build:gradle:1.+' // Android gradle 1.+ is for Gradle 2.+ projects
```

Comment:
OK well the comment is not accurate. Version 1.0.0 cannot be used with
Gradle 2.0 or 2.1 and it most certainly will be compatible with 3.x.
Previous versions worked across the 1.x and 2.x divide.
On Jan 9, 2015 5:31 PM, "Greg Benson" <notifications@github.com> wrote:

> In sample-app/build.gradle
> <https://github.com/ReactiveX/RxAndroid/pull/121#discussion-diff-22754595>
> :
>
> > @@ -1,7 +1,8 @@
> >  buildscript {
> >      repositories { jcenter() }
> >      dependencies {
> > -        classpath 'com.android.tools.build:gradle:0.12.+' // 0.12.x is for Gradle 1.12 projects
> > +        classpath 'com.android.tools.build:gradle:1.+' // Android gradle 1.+ is for Gradle 2.+ projects
>
> Comment is not redundant because otherwise it's unclear how then android
> gradle version maps to standard gradle instals.
> Understand the wildcard dep, fixing...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ReactiveX/RxAndroid/pull/121/files#r22754595>.
>